File: Assets/LevelPlay/Runtime/Platforms/Android/IUnityInterstitialAdListener.cs
namespace Unity.Services.LevelPlay
{
    interface IUnityInterstitialAdListener
    {
        void onAdLoaded(string adInfo);
        void onAdLoadFailed(string error);
        void onAdDisplayed(string adInfo);
        void onAdDisplayFailed(string error, string adInfo);
        void onAdClosed(string adInfo);
        void onAdClicked(string adInfo);
        void onAdInfoChanged(string adInfo);
    }
}

File: Assets/LevelPlay/Runtime/Plugins/Android/IUnityInterstitialAdListener.java
package com.ironsource.unity.androidbridge;

interface IUnityInterstitialAdListener {
    void onAdLoaded(String adInfo);
    void onAdLoadFailed(String error);
    void onAdDisplayed(String adInfo);
    void onAdDisplayFailed(String error, String adInfo);
    void onAdClosed(String adInfo);
    void onAdClicked(String adInfo);
    void onAdInfoChanged(String adInfo);
}
